reference/general/README.md

```diff
@@ -21,13 +19,11 @@ description: >-
 
 * [**Lotus**](https://github.com/filecoin-project/lotus) - _**Recommended**_ most advanced implementation and [security audited](https://spec.filecoin.io/#section-intro.implementations-status), in GoLang.
 * Other implementations (In progress):
```

Comment:
I would at the very least move Venus out of the (In progress) category or just remove in progress.

Reply:
Resolved